.travis.yml

```diff
@@ -191,6 +191,10 @@ jobs:
       script: ./tools/travis_controller.sh misc
       env: PROFILE="-Dhadoop.version=2.8.3 -Pinclude-kinesis -Dinclude_hadoop_aws -Dscala-2.12"
       name: misc - scala 2.12
+    - if: type = cron
+      script: ./tools/travis_controller.sh connector_hive_1
+      env: PROFILE="-Dhadoop.version=2.8.3 -Pinclude-kinesis -Dinclude_hadoop_aws -Dscala-2.12"
```

Comment:
should we use hadoop 2.6.5 which hive 1.2.1 is built on,  rather than 2.8.3?

Reply:
We can't because that'll be a different profile. And profile is used to get the Flink cache between jobs. Since we're using a higher version, hopefully that'll be fine.

Reply:
I would suggest to add `-Phive-1.2.1` to every scala 2.12 profile; then we get the test coverage without requiring another profile that uses funky hacks.